contrib/libskk/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "libskk"
+pkgver = "1.0.5"
+pkgrel = 0
+build_style = "configure"
+configure_args = ["--prefix=/usr"]
+make_cmd = "gmake"
+hostmakedepends = [
+    "gettext-devel",
+    "gmake",
+    "gobject-introspection",
+    "pkgconf",
+    "vala",
+]
+makedepends = [
+    "json-glib-devel",
+    "libgee-devel",
+    "libxkbcommon-devel",
+    "vala-devel",
+]
+depends = ["skkdict"]
+pkgdesc = "Japanese input method library"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "GPL-3.0-or-later"
+url = "https://github.com/ueno/libskk"
+source = f"{url}/releases/download/{pkgver}/libskk-{pkgver}.tar.xz"
+sha256 = "b295eb9c248e353750c96474b2b55c4e627524fc6dc79439ff56f73ba09c3949"
+
+
+@subpackage("libskk-devel")
+def _devel(self):
+    return self.default_devel()
+
+
+tool_flags = {
```

Comment:
wrong place, should be in the main metadata block or at very least not after any subpackage definitions